packages/shorebird_cli/test/src/commands/patch_new/patch_new_command_test.dart

```diff
@@ -136,6 +140,8 @@ void main() {
         () => artifactManager.downloadFile(any()),
       ).thenAnswer((_) async => File(''));
 
+      when(() => cache.updateAll()).thenAnswer((_) async => {});
```

Comment:
should we also verify that `cache.updateAll` is called later in a test?

Reply:
Good call, done